src/Objs/Job/JobComponent.js

```diff
@@ -1,17 +1,94 @@
+import formatDate from 'utils/formatDate';
+import SchemaDotOrg from 'Components/SchemaDotOrg';
+import InPlaceEditingPlaceholder from 'Components/InPlaceEditingPlaceholder';
+
 Scrivito.provideComponent('Job', ({ page }) => {
   return (
     <div>
       <section className="bg-white">
         <div className="container">
+          <Date text="" date={ page.get('datePosted') } />
           <Scrivito.ContentTag tag="h1" className="h2" content={ page } attribute="title" />
-          <h2 className="h4">
-            <i className="fa fa-map-marker fa-lg" aria-hidden="true" title="location" />
+          <JobLocation job={ page } />
+          <h3 className="h5">
```

Comment:
Location and "Tags" are already its own component. I suggest to also make the date its own location.

I would also rename the components:

* `JobLocation`
* `JobValidThrough`
* `JobEmploymentTypes`